reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -62,7 +62,7 @@ <h1 id="liveupdate-title">${thing.event.title}</h1>
     </ul>
 
     % if len(thing.contributors) > CONTRIBUTOR_CUTOFF:
-    <a href="/live/${c.liveupdate_event._id}/contributors" class="more-contributors">${unsafe(_("&hellip; and %(count)s more &raquo;") % dict(count=len(thing.contributors) - CONTRIBUTOR_CUTOFF))}</a>
+    <a href="/live/${c.liveupdate_event._id}/contributors" class="more-contributors">${_("… and %(count)s more ⇒") % dict(count=len(thing.contributors) - CONTRIBUTOR_CUTOFF)}</a>
```

Comment:
Yep, so it could benefit from the existing translation at https://github.com/reddit/reddit/blob/master/r2/r2/public/static/js/uibase.js#L74. I can keep it the same if you want